src/components/Pane.jsx

```diff
@@ -0,0 +1,47 @@
+var React = require('react'),
+    classNames = require('classnames'),
+    style = require('./component-playground.less');
+
+export default React.createClass({
+  displayName: 'Pane',
+
+  propTypes: {
+    split: React.PropTypes.oneOf(['vertical', 'horizontal']),
+    children: React.PropTypes.object.isRequired
+  },
+
+  getInitialState: function() {
+    return {};
+  },
+
+  render: function() {
+    var classes = {};
+    classes[style[this.props.split]] = true;
+    classes[style['pane']] = true;
+    classes = classNames(classes);
+
+    var split = this.props.split || 'vertical';
+
+    var styles = {
+      flex: 1,
+      position: 'relative',
+      outline: 'none',
+      overflow: 'none'
+    };
+    if (this.state.size) {
+      if (split === 'horizontal') {
+        styles.height = this.state.size;
+        styles.display = 'flex';
+      } else {
+        styles.width = this.state.size;
+      }
+      styles.flex = 'none';
+    }
+
+    return (
+    <div className={classes} style={styles}>
```

Comment:
Original `component-playground.jsx` seems to have the `return ()` styling in place. We'll also need to change it there if we commit to this kind of styling. @valentin-radulescu-hs